src/Sylius/Component/Core/Promotion/Action/UnitFixedDiscountPromotionActionCommand.php

```diff
@@ -78,4 +87,13 @@ private function setUnitsAdjustments(OrderItemInterface $item, int $amount, Prom
             );
         }
     }
+
+    private function applyAdditionalItemFilters(array $filteredItems, array $configuration): array
+    {
+        foreach($this->additionalItemFilters as $additionalItemFilter) {
+            $filteredItems = $additionalItemFilter->filter($filteredItems, $configuration);
+        }
+
+        return $filteredItems;
+    }
```

Comment:
It's worth making a separate service that would be reusable instead of having the same private methods in many places.